src/lib/datepicker/calendar.ts

```diff
@@ -44,7 +44,20 @@ import {MatMonthView} from './month-view';
 import {MatMultiYearView, yearsPerPage, yearsPerRow} from './multi-year-view';
 import {MatYearView} from './year-view';
 import {Directionality} from '@angular/cdk/bidi';
+import {ComponentPortal, ComponentType, Portal} from '@angular/cdk/portal';
 
+/**
+ * Default header of a [MatCalendar].
```

Comment:
`/** Default header for MatCalendar */`

Reply:
done in 0faefc46a68fb3a15f91945858faad7a399c8c8a